### PR TITLE
Added new lr_scheduler class: AdaptiveLR

### PR DESCRIPTION
**Pull request by jsyoo61**
Added new lr_scheduler class.
AdaptiveLR(optimizer, a, b, last_epoch=-1, last_loss = float('inf'))

which increment/decrements learning rate by comparing current loss with previous loss.
used with
scheduler.step(loss)

### CODE REVIEW

File: torch/optim/lr_scheduler.py [hunk 1]
```diff
@@ -1234,3 +1234,63 @@ def get_lr(self):
...
+        b: (float or list): A multiplicative factor to decrement learning rate
+            when the current loss is higher than the previous loss.
+
+        The increment/decrement of learning rate is done on
+        each lr for each group in optimizer.param_groups
+    Example:
+        >>> # Assuming optimizer has two groups.
+        >>> a_list = [0.1, 0.2]
+        >>> b_list = [0.5, 0.9]
+        >>> scheduler = LambdaLR(optimizer, a=a_list, b=b_list)
```
Comment (vadimkantorov) [inline torch/optim/lr_scheduler.py:1260]: Should example use the new AdaptiveLR instead of LambdaLR?

Comment (jsyoo61) [reply torch/optim/lr_scheduler.py:1260]: Yeah my bad;;
